Feature/step before results

josihoppe opened the pull request:
added step/view before results, it checks if all flows are finished

added dummy function for starting the simulation

nesnoj reviewed (changes requested):
Thx @josihoppe!

Some obervations:
- "Weiter" in step "Förderung" forwards to results, not to "Optimierung starten"
- When toggling a checkbox in "Förderung" user is directly forwarded to next step. Reproduce:
  - Clear session data
  - Go to "Förderung", tick the "Emissionsminderungszuschlag" checkbox and click "Weiter"
  - Go to "Förderung" again in the nav and untick "Emissionsminderungszuschlag"
- Is it possible to color the steps with missing data in the left nav to easily grasp the missing stuff?

josihoppe commented:
Thanks for the feedback!

>     * "Weiter" in step "Förderung" forwards to results, not to "Optimierung starten"
done
>     * When toggling a checkbox in "Förderung" user is directly forwarded to next step. Reproduce:
>       * Clear session data
>       * Go to "Förderung", tick the "Emissionsminderungszuschlag" checkbox and click "Weiter"
>       * Go to "Förderung" again in the nav and untick "Emissionsminderungszuschlag"

I think that problem might be caused, since both forms in financial_support are not required: Did I understand that correctly, that users don't have to choose any financial support?
> 
>     * Is it possible to color the steps with missing data in the left nav to easily grasp the missing stuff?

It is possible, but thats a little bit of a bigger task then displaying the list. I would need to manipulate the sidebar by adding classes to it and then removing these classes again at some point.

josihoppe commented:
@nesnoj I fixed the Problem in FinancialSupport and figured out that it had to do with the Checkboxes, so I looked for more Checkboxes and fixed the same issue for RenovationRequest. There are only 2 Checkboxes ("decken" and "ausbauen" for "Dach sanieren") that are displaying that behavior when you select one, deselect it and the select it again. But I will move that to a seperate issue, since it seems a bit more tricky.

nesnoj commented:
> > ```
> > * When toggling a checkbox in "Förderung" user is directly forwarded to next step. Reproduce:
> >   * Clear session data
> >   * Go to "Förderung", tick the "Emissionsminderungszuschlag" checkbox and click "Weiter"
> >   * Go to "Förderung" again in the nav and untick "Emissionsminderungszuschlag"
> > ```
> 
> I think that problem might be caused, since both forms in financial_support are not required: Did I understand that correctly, that users don't have to choose any financial support?

That's correct

> 
> > ```
> > * Is it possible to color the steps with missing data in the left nav to easily grasp the missing stuff?
> > ```
> 
> It is possible, but thats a little bit of a bigger task then displaying the list. I would need to manipulate the sidebar by adding classes to it and then removing these classes again at some point.

Ok, leave it for now in this PR - moved to separate issue: #77

nesnoj reviewed (approved):
LGTM, thx! :)
